runtime/src/main/kotlin/kotlin/Exceptions.kt

```diff
@@ -166,13 +166,6 @@ public actual open class NumberFormatException : IllegalArgumentException {
     actual constructor(message: String?) : super(message)
 }
 
-public open class IllegalCharacterConversionException : IllegalArgumentException {
```

Comment:
It's possible to leave it here deprecated.

Reply:
+1